gcp-observability/src/main/java/io/grpc/gcp/observability/logging/GcpLogSink.java

```diff
@@ -133,6 +119,14 @@ public void write(GrpcLogRecord logProto) {
     }
   }
 
+  Logging createLoggingClient() {
```

Comment:
nit: the location of this method changed which seems unnecessary. Or did it change because of the access modifier?

Reply:
I modified it in order to keep public methods before package private and private methods. I also notice that in order to achieve that  `close()` needs to be in put in top of the class along with other public methods.
If you think this is not necessary, I will undo this change.

Reply:
No it's okay, not a big deal.